### DISCUSSION TITLE
Cannot adjust brightness after enable dGPU

### DISCUSSION THREAD

**Discussion by JerrardH** [seerge/g-helper#2039] [Q&A]
Device: ROG flow z13 RTX4050
I set gpu mode to optimal and let it to enable 4050 when plugged in to power, and use igpu on battery. I have no issue adjusting the brightness while using the intel graphics card or in the ultimate mode. There is a high chance (not always) that after I plugged in and auto enabled dgpu, I cannot change the screen brightness. At this point, restart the computer would work. Also manually set to eco mode would be able to change brightness, but if I manually switching back to standard the brightness is still not working. Any ideas?

**Comment by seerge** [comment]
@JerrardH hello, please check this topic that [addresses your issue](https://github.com/seerge/g-helper/discussions/1042)
